tests/Core/Ruleset/PopulateTokenListenersTest.php

```diff
@@ -62,11 +62,16 @@ public function testSniffWhereRegisterDoesNotReturnAnArrayThrowsException()
         $config   = new ConfigDouble(["--standard=$standard"]);
 
         $sniffClass = 'Fixtures\\TestStandard\\Sniffs\\InvalidSniffs\\RegisterNoArraySniff';
-        $message    = "ERROR: Sniff $sniffClass register() method must return an array";
+        $message    = "ERROR: The sniff $sniffClass register() method must return an array.".PHP_EOL.PHP_EOL;
```

Comment:
Possibly out of scope, but given we're already changing this text: should we associate the method name more closely with its class?
```suggestion
        $message    = "ERROR: The sniff $sniffClass::register() method must return an array.".PHP_EOL.PHP_EOL;
```

Reply:
Fair point. Done.